extensions/amp-carousel/0.1/slidescroll.js

```diff
@@ -598,8 +614,8 @@ export class AmpSlideScroll extends BaseSlides {
    * @private
    */
   hideRestOfTheSlides_(indexArr) {
-    const noOfSlides_ = this.noOfSlides_;
-    for (let i = 0; i < noOfSlides_; i++) {
+    const totalNumSlides = this.slides_.length;
```

Comment:
this.noOfSlides_ should always be the source of truth so we should only use that.